libs/langchain/langchain/agents/agent.py

```diff
@@ -53,6 +54,7 @@
 logger = logging.getLogger(__name__)
 
 
+@deprecated("0.1.0", removal="0.2.0")
```

Comment:
will this raise an error when using RunnableAgent since that inherits from this?

Reply:
believe it doesn't but will double check